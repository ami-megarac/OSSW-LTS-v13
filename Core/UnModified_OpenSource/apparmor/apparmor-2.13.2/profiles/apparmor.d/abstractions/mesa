# vim:syntax=apparmor
# Rules for Mesa implementation of the OpenGL API

  # System files
  /dev/dri/ r, # libGLX_mesa.so calls drmGetDevice2()

  # User files
  owner @{HOME}/.cache/ w, # if user clears all caches
  owner @{HOME}/.cache/mesa_shader_cache/ w,
  owner @{HOME}/.cache/mesa_shader_cache/index rw,
  owner @{HOME}/.cache/mesa_shader_cache/??/ w,
  owner @{HOME}/.cache/mesa_shader_cache/??/* rwk,

