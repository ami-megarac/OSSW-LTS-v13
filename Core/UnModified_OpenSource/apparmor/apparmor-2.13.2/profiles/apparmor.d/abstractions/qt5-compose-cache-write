# vim:syntax=apparmor
# Allow writing cache for Qt5 "platforminputcontexts" plugins

  # User files

  owner @{HOME}/.cache/qt_compose_cache_{little,big}_endian_* rw,

