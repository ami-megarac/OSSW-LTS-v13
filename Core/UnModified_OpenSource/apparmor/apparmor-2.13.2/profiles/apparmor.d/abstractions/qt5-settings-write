# vim:syntax=apparmor
# Allow writing shared settings for Qt-based applications

  # User files

  owner @{HOME}/.config/#[0-9]* rw,
  owner @{HOME}/.config/QtProject.conf rw,
  owner @{HOME}/.config/QtProject.conf.?????? l -> @{HOME}/.config/#[0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9],
  owner @{HOME}/.config/QtProject.conf.?????? rw, # for temporary files like QtProject.conf.Aqrgeb
  owner @{HOME}/.config/QtProject.conf.lock rwk,

