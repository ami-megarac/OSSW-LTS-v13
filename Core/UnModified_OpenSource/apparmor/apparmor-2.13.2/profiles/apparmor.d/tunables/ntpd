# Last Modified: Thu Aug  2 14:37:03 2007
# ------------------------------------------------------------------
#
#    Copyright (C) 2002-2005 Novell/SUSE
#
#    This program is free software; you can redistribute it and/or
#    modify it under the terms of version 2 of the GNU General Public
#    License published by the Free Software Foundation.
#
# ------------------------------------------------------------------

#Add your ntpd devices here eg. if you have a DCF clock
# @{NTPD_DEVICE}=/dev/ttyS*
@{NTPD_DEVICE}="/dev/tty10"
