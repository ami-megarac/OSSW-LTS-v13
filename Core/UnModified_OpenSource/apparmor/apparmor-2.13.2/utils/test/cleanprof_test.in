# A simple test comment which will persist
#include <tunables/global>

		alias /foo    ->    /bar	,
  abi  <abi/4.19>    ,

/usr/bin/a/simple/cleanprof/test/profile {
	# Just for the heck of it, this comment wont see the day of light
	#include <abstractions/base>

    capability sys_admin,
    audit capability,

    change_profile -> /bin/foo,
    change_profile,

    network inet stream,
               abi  "abi/4.20"    ,
    network stream,

	#Below rule comes from abstractions/base
	allow /usr/share/X11/locale/**  r,
	allow /home/*/** r,

    ptrace tracedby peer=/bin/strace,
    ptrace tracedby,
    unix (receive) type=dgram,

    dbus send bus=session,
    dbus send bus=session peer=(label=foo),

    set rlimit nofile <= 256,
    set rlimit nofile <= 64,

    signal set=(hup int quit ill trap abrt)
             set=(bus,fpe,,,kill,usr1)
                      set=segv set=usr2 set=pipe set=alrm set=term set=stkflt set=chld,
    signal set=(hup int quit),

    ^foo {
            /etc/fstab r,
        capability dac_override,
        }

    ^foo, # hat declarations are obsolete and will be removed when aa-cleanprof or aa-logprof writes the profile


	link subset /alpha/beta -> /tmp/**,

	allow /home/foo/bar r,
	allow /home/foo/** w,
}

/usr/bin/other/cleanprof/test/profile {
	# This one shouldn't be affected by the processing
	# However this comment will be wiped, need to change that
	allow /home/*/** rw,
	allow /home/foo/bar r,
}
