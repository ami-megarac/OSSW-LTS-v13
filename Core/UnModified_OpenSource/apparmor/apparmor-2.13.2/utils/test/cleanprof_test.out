abi  <abi/4.19>    ,

alias /foo -> /bar,

#include <tunables/global>

# A simple test comment which will persist


/usr/bin/a/simple/cleanprof/test/profile {
  abi  "abi/4.20"    ,

  #include <abstractions/base>

  set rlimit nofile <= 256,

  audit capability,

  network stream,

  dbus send bus=session,

  signal set=(abrt alrm bus chld fpe hup ill int kill pipe quit segv stkflt term trap usr1 usr2),

  unix (receive) type=dgram,

  link subset /alpha/beta -> /tmp/**,

  allow /home/*/** r,
  allow /home/foo/** w,

  change_profile,


  ^foo {
    capability dac_override,

    /etc/fstab r,

  }
}
/usr/bin/other/cleanprof/test/profile {
  allow /home/*/** rw,
  allow /home/foo/bar r,

}
